src/main/java/com/world/entertainment/worldentertainment/entity/Customer.java

```diff
@@ -12,17 +12,15 @@
     @GeneratedValue(strategy = GenerationType.IDENTITY)
     private int id;
 
-    private String name;
-
     @OneToMany(mappedBy = "customer")
     private Set<Entertainment> entertainments;
 
-    private boolean isActive;
+    private String name;
+    private String email;
 
+    private boolean isActive;
     private Date dateCreate;
```

Comment:
Let's rename this fields as `deleteAt`, `updateAt` and `CreateAt`. We use this style in other classes.